myapp/app/controllers/admin/users_controller.rb

```diff
@@ -37,6 +38,11 @@ def show
   end
 
   def destroy
+    if @user.admin? && User.where(role: "admin").count <= 1
+      flash[:alert] = "You cannot delete the only admin user."
```

Comment:
```suggestion
    if @user.admin? && User.where(role:'admin').count <= 1
      flash[:alert] = 'You cannot delete the only admin user.'
```
Ruby strings are basically enclosed in single quotes.
(using fablicop is recommended.)

Reply:
@hayashiR1 Got it! Thanks for your advice.